src/js/gi/components/heading/Breadcrumbs.jsx

```diff
@@ -30,15 +24,12 @@ class Breadcrumbs extends React.Component {
       crumbs.push(<span key="profile"><strong>{this.props.profileName || 'Profile'}</strong></span>);
```

Comment:
Don't you need to remove this too? When showing the profile, the profile is currently in prod the "active" crumb:

![screen shot 2017-08-04 at 4 33 57 pm](https://user-images.githubusercontent.com/6646098/28987874-dc8cc2bc-7932-11e7-9be9-053cb9f906b9.png)